grpc-server/src/main/java/com/netflix/conductor/grpc/server/service/EventServiceImpl.java

```diff
@@ -0,0 +1,98 @@
+package com.netflix.conductor.grpc.server.service;
+
+import com.google.protobuf.Empty;
+
+import com.netflix.conductor.core.events.EventProcessor;
+import com.netflix.conductor.core.events.EventQueues;
+import com.netflix.conductor.grpc.EventServiceGrpc;
+import com.netflix.conductor.grpc.EventServicePb;
+import com.netflix.conductor.grpc.ProtoMapper;
+import com.netflix.conductor.proto.EventHandlerPb;
+import com.netflix.conductor.service.MetadataService;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EventServiceImpl extends EventServiceGrpc.EventServiceImplBase {
+    private static final Logger logger = LoggerFactory.getLogger(EventServiceImpl.class);
+    private static final ProtoMapper protoMapper = ProtoMapper.INSTANCE;
+    private static final GRPCHelper grpcHelper = new GRPCHelper(logger);
```

Comment:
use all upper case names for constants

Reply:
The existing server implementation (i.e. in the Jersey module) already has a `logger` static field with a lowercase name, which is what I brought over here. Would you like me to rename these fields to uppercase and also the ones in the jersey module, for consistency?

Reply:
That would be great.